docs/content.zh/docs/dev/table/sql/truncate.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "TRUNCATE TABLE 语句"
+weight: 8
+type: docs
+aliases:
+- /zh/dev/table/sql/truncate.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<a name="truncate-statements"></a>
+
+# TRUNCATE TABLE 语句
+
+DESCRIBE 语句用于删除表中的全部数据，但不会删除表本身。TRUNCATE TABLE 语句目前仅支持使用在Batch模式。
```

Comment:
```suggestion
TRUNCATE TABLE 语句用于删除表中的全部数据，但不会删除表本身。TRUNCATE TABLE 语句目前仅支持在Batch 模式下使用。
```

Reply:
Please add the attention just like [`DELETE` statement ](https://nightlies.apache.org/flink/flink-docs-master/zh/docs/dev/table/sql/delete/) in FLINK.
```
注意 目前, DELETE 语句仅支持批模式, 并且要求目标表实现了 [SupportsRowLevelDelete ](https://github.com/apache/flink/blob/master/flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/sink/abilities/SupportsRowLevelDelete.java)接口。 如果在一个没有实现该接口的表上执行 DELETE，则会抛异常。
```